packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -81,16 +82,19 @@ describe('Withdraw ERC20 Token', () => {
             .should('be.visible')
             .click({ scrollBehavior: false })
 
-          // Select the LINK token
-          cy.findByText('ChainLink Token').click({ scrollBehavior: false })
+          // Select the WETH token
+          cy.findAllByText('WETH').first().click({ scrollBehavior: false })
 
-          // LINK token should be selected now and popup should be closed after selection
+          // WETH token should be selected now and popup should be closed after selection
           cy.findByRole('button', { name: 'Select Token' })
             .should('be.visible')
-            .should('have.text', 'LINK')
+            .should('have.text', 'WETH')
         })
     })
 
+    // TODO: WETH deposit takes time to be refected in the balance.
+    // In theory there's a chance it won't show here before tests.
+    // Fix: await for deposit result and continue tests.
```

Comment:
I think we can get rid of this comment as we're not waiting for the balance to be updated right now, just that it's initiated